Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1765,25 +1767,27 @@ def malware_query_command(client: Client, args: dict):
     malware_status = str(args.get('status'))
     time_stamp = str(args.get('timestamp'))
     limit_range = arg_to_number(args.get('limit'))
-    if limit_range > 0:
-        filter_response = malware_query_filter(client, needs_attention, malware_type, malware_status, time_stamp, limit_range)
-        return CommandResults(raw_response=filter_response)
+    if limit_range:
+        if limit_range > 0:
```

Comment:
Hi @JasBeilin ,
The extra If is added to avoid negative values.
If the limit_range is negative, the condition if limit_range will be true and will continue further. So that's why we have added an extra If.